src/Pages/ChannelRequests.razor

```diff
@@ -141,15 +141,15 @@
                 </DisplayTemplate>
                 <EditTemplate>
                     <Validation Validator="@ValidationHelper.ValidateChannelCapacity" @ref="_capacityValidation">
-                        <NumericPicker TValue="decimal" @bind-Value="@_amount" Min="0" Decimals="8" Step="0.00001m" CurrencySymbol=" BTC" CurrencySymbolPlacement="CurrencySymbolPlacement.Suffix" Disabled="@(SelectedUTXOs.Count > 0)">
-                            <Feedback>
-                                <ValidationError/>
-                            </Feedback>
-                        </NumericPicker>
+                        <NumericPicker TValue="decimal" @bind-Value="@_amount" Min="@_minimumChannelCapacity" Max="Constants.MAXIMUM_WITHDRAWAL_BTC_AMOUNT" Decimals="8" Step="0.00001m" CurrencySymbol=" BTC" CurrencySymbolPlacement="CurrencySymbolPlacement.Suffix" Disabled="@(SelectedUTXOs.Count > 0)"/>
                         <FieldHelp>
-
-                            @($"Amount in Satoshis. Minimum {_minimumChannelCapacity:f8}. Current amount: {Math.Round(PriceConversionService.SatToUsdConversion(new Money(_amount, MoneyUnit.BTC).Satoshi, _btcPrice), 2)} USD")
-
+                              @{
+                                  decimal amountToShow = _amount < Constants.MAXIMUM_WITHDRAWAL_BTC_AMOUNT
+                                      ? _amount
+                                      : Constants.MAXIMUM_WITHDRAWAL_BTC_AMOUNT;
+                                  decimal convertedAmount = Math.Round(PriceConversionService.SatToUsdConversion(new Money(amountToShow, MoneyUnit.BTC).Satoshi, _btcPrice), 2);
+                              }
+                            @($"Amount in Satoshis. Minimum {_minimumChannelCapacity:f8}. Current amount: {convertedAmount} USD")
```

Comment:
In the Channel Request modal, the error message stating that the channel capacity should be lower than 0.16777215 BTC doesn't show up anymore.

<img width="967" alt="Screenshot 2023-10-05 at 09 59 38" src="https://github.com/Elenpay/NodeGuard/assets/39995243/5b36056e-1f16-49e9-ac07-3e4b579ca5bb">

Reply:
Fixed.